api/support/verify_state.py

```
@@ -0,0 +1,294 @@
+    # 2. Proof from Chainstack.
+    try:
+        proof = await fetch_account_proof(session, chainstack_url, addr_hex, block_hex)
+    except Exception as e:
+        logging.warning(f"verify_state: proof fetch failed for {chain}: {e}")
+        return [_format_verifier_status_line(chain, STATUS_PROOF_UNAVAILABLE, ts_ns)]
+
+    # 3. Local MPT verification.
+    try:
+        balance = verify_account_proof(addr_bytes, proof, anchor)
+    except ProofError as e:
+        logging.error(f"verify_state: proof math invalid for {chain}: {e}")
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)